ups/ts_athexapod.table

```diff
@@ -2,9 +2,6 @@
 # - Any package whose API is used directly should be listed explicitly.
 # - Common third-party packages can be assumed to be recursively included by
 #   the "sconsUtils" package.
-setupRequired(ts_salobj)
-setupRequired(ts_idl)
-setupRequired(ts_config_attcs)
```

Comment:
Why did you remove these lines? ATHexapod still depends on those packages, right?